docs/source/install.rst

```diff
@@ -6,52 +6,63 @@ Install
 Currently installation is supported by constructing a conda_ environment. You need to be running
 a conda_ package (eg. miniconda, anaconda, etc) to install autometa.
 
-Conda environment creation:
-
-.. code-block:: bash
-
-    conda create -n autometa "python>=3.7"
-    conda install -n autometa -c bioconda -c conda-forge \
-        biopython \
-        pandas \
-        tqdm \
-        tsne \
-        numpy \
-        scikit-learn \
-        scikit-bio \
-        samtools \
-        bedtools \
-        bowtie2 \
-        hmmer \
-        prodigal \
-        diamond \
-        ndcctools \
-        parallel \
-        requests \
-        umap-learn \
-        hdbscan
+Conda installation
+==================
 
+#. Install miniconda_
+#. Create a new environment ``conda create -n autometa "python>=3.7"``
+#. Install autometa ``conda install -c conda-forge -c bioconda autometa --yes``
+#. Actiavate autometa envoirnonment ``conda activate autometa``
 
-.. _conda: https://docs.conda.io/en/latest/
+Docker image
+============
+
+You can also run Autometa using a prebuild Docker image. 
+
+#. Install Docker_
+#. Run the following commands
+
+    .. code-block:: bash
+
+        git clone https://github.com/KwanLab/Autometa.git
+        docker pull jasonkwan/autometa:latest
```

Comment:
1. There is no need to clone the repository if the user just wants to use the docker image.
2. Also no need to run docker pull as docker recognize if an image is not present/available.

User should be able to just run 

```bash
docker run jason-c-kwan/autometa:dev
docker run jason-c-kwan/autometa:latest
... whatever other tags we plan on having available
```

and docker will retrieve the image if they do not already have it.